examples/power_spectra_example.py

```diff
@@ -0,0 +1,44 @@
+"""
+===============================
+Power Spectrum of a TimeSeries
+===============================
+
+An example showing how to estimate the power spectrum of a TimeSeries.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import matplotlib.pyplot as plt
+import pandas as pd 
+from scipy import signal
+import astropy.units as u
+
+import sunpy.timeseries
+from sunpy.data.sample import RHESSI_TIMESERIES
+
+###############################################################################
+# Let's first load a TimeSeries from sample data.
+# Input data contains 9 columns, which are evenly sampled with a time
+# step of 4 seconds. So, the data is suitable for signal analysis.
+ts = sunpy.timeseries.TimeSeries(RHESSI_TIMESERIES)
+
+###############################################################################
+# We now use SciPy's periodogram to estimate the power spectra of the first 
```

Comment:
I think that `~scipy.signal.lombscargle` should work as a link to the documentation

Reply:
Ohh! Thats quite useful. Thanks a lot @nabobalis